science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+<figure class="figure-img">
+<img src="./ppa_test_result_inzidenz_pcr_2-1_en.png" class="figure-img img-fluid" alt="Figure 15: Relationship between local 7-day incidence and positivity rate after risk notification and incidence areas (PCR)."/>
+<figcaption aria-hidden="true">Figure 15: Relationship between local 7-day incidence and positivity rate after risk notification and incidence areas (PCR).</figcaption>
+</figure>
+
+We can also summarise this relationship in a continuous figure (Fig. 16A) and also give a linear approximation of this relationship (Fig.16B).
+
+<figure class="figure-img">
+<img src="./ppa_test_result_inzidenz_pcr_3-1_en.png" class="figure-img img-fluid" alt="Figure 16: Relationship between local 7-day incidence and positivity rate after risk notification, smoothed and with confidence interval, in (A) nonlinear and (B) linear approximation (PCR)."/>
+<figcaption aria-hidden="true">Figure 16: Relationship between local 7-day incidence and positivity rate after risk notification, smoothed and with confidence interval, in (A) nonlinear and (B) linear approximation (PCR).</figcaption>
+</figure>
+
+The non-linear curves (left) flatten with increasing incidence. Presumably because in times of high incidence, contact-restricting measures or a lockdown were increasingly introduced and therefore, on the one hand, the contact rates were lower and, on the other hand, tests that were needed for meetings and for which a negative result was already expected were performed less frequently.
+
+For a red warning, the dependency is only about half as strong as in the case of a lower risk without risk exposure. The reason for this is probably that about half of the population actively uses the CWA, and the risk of contracting the disease (undetected by the CWA) from a person who does not use the CWA increases with increasing incidence. It can be assumed that with more numerous use of the CWA, this dependence of the positivity rate on incidence would be further reduced. As mentioned above, if the rate of utilisation were even higher, the uncertainty in the assessment of risk exposures would be reduced even more.
+
+The positivity rate at lower risk without a risk exposure is greater with increasing incidence than the positivity rate at increased risk. This is due on the one hand to the fact that the dependence on incidence is twice as high here, and on the other hand to the fact that there is presumably a strong pre-selection here through rapid antigen tests (performed officially or at home). Only if this was positive was it carried out. Further studies are needed to understand these relationships more thoroughly. For example, an event-related user survey could be used to ascertain the extent of preselection. An agent-based simulation that models different individual behaviours could also provide possible explanations.
```

Comment:
"Only if this was positive was it carried out."
=> 
"Only if this was positive was a PCR test carried out."
otherwise it is a bit unclear

Reply:
Thanks for your suggestion, it has been implemented in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5